src/Nest/Mapping/Types/Geo/GeoPoint/GeoPointAttribute.cs

```diff
@@ -8,6 +8,10 @@ public class GeoPointAttribute : ElasticsearchDocValuesPropertyAttributeBase, IG
 
 		bool? IGeoPointProperty.IgnoreMalformed { get; set; }
 
-		public bool IgnoreMalformed { get { return Self.IgnoreMalformed.GetValueOrDefault(); } set { Self.IgnoreMalformed = value; } }
+		bool? IGeoPointProperty.IgnoreZValue { get; set; }
+		GeoLocation IGeoPointProperty.NullValue { get; set; }
+
+		public bool IgnoreMalformed { get => Self.IgnoreMalformed.GetValueOrDefault(); set => Self.IgnoreMalformed = value; }
+		public bool IgnoreZValue { get => Self.IgnoreZValue.GetValueOrDefault(); set => Self.IgnoreZValue= value; }
```

Comment:
I think this should be `Self.IgnoreZValue.GetValueOrDefault(true)`, in line with the default value in Elasticsearch